build/bundles.js

```diff
@@ -2,18 +2,43 @@ module.exports = {
   "bundles": {
     "scripts/app-build": {
       "includes": [
+        "[**/*.js]",
```

Comment:
yes. i think all that's not in other bundles or so. anyways. jspm_packages are not in app-build.js in the end